src/org/openstreetmap/josm/plugins/osmobjinfo/OSMObjInfoActions.java

```diff
@@ -64,26 +72,75 @@ public static void openinBrowserChangesetMap(String idChangeset) {
             openInBrowser("https://osmcha.mapbox.com/".concat(idChangeset));
         }
     }
-
-    public static void copyIdobj(String typeObj, String idobj) {
-        if (typeObj != null && !idobj.isEmpty()) {
-            String linkobjid = "https://www.openstreetmap.org/" + typeObj + "/" + idobj;
-            StringSelection selection = new StringSelection(linkobjid);
-            Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
-            clipboard.setContents(selection, selection);
-            new Notification(tr("Copy: " + linkobjid)).setIcon(JOptionPane.INFORMATION_MESSAGE).setDuration(Notification.TIME_SHORT).show();
+    
+    public static void copyIdobj(List<AllOsmObjInfo> osmObjInfo) {
+        if (osmObjInfo == null || osmObjInfo.isEmpty()) return;
+        String linkobjid = "";
+        StringBuilder builder = new StringBuilder();
+        for (AllOsmObjInfo object : osmObjInfo) {
+            builder.append("https://www.openstreetmap.org/")
+                .append(object.typeObj).append("/").append(object.idObject);
+            if (osmObjInfo.indexOf(object) < osmObjInfo.size() - 2) {
+                builder.append(", ");
+            }
+            if (osmObjInfo.indexOf(object) == osmObjInfo.size() - 2) {
+                if (osmObjInfo.size() == 2) {
+                    builder.append(" ");
+                } else {
+                    builder.append(", ");
+                }
+                builder.append(tr("and"));
+                builder.append(" ");
+            }
         }
+        linkobjid = builder.toString();
+        StringSelection selection = new StringSelection(linkobjid);
+        Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
+        clipboard.setContents(selection, selection);
+        new Notification(tr(COPY, linkobjid)).setIcon(JOptionPane.INFORMATION_MESSAGE).setDuration(Notification.TIME_SHORT).show();
+    }
+    
+    public static void copyRemoteControl(List<AllOsmObjInfo> osmObjInfo) {
+    	if (osmObjInfo == null || osmObjInfo.isEmpty()) return;
+    	StringBuilder builder = new StringBuilder();
+        builder.append("http://127.0.0.1:8111/load_and_zoom?");
```

Comment:
instead of hardcoding it, it should be retrieved from JOSM core

Reply:
Fair enough. I'll make the change.